publisher/publisher.go

```diff
@@ -369,3 +391,37 @@ func CreateTestingSignedSCT(req []string, k *ecdsa.PrivateKey, precert bool, tim
 	jsonSCT, _ := json.Marshal(jsonSCTObj)
 	return jsonSCT
 }
+
+func (pub *Impl) ProbeLogs() {
+	wg := new(sync.WaitGroup)
+	for _, log := range pub.ctLogsCache.LogURIs() {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			c := http.Client{
+				Timeout: time.Minute*2 + time.Second*30,
```

Comment:
Did you pick this timeout based on the ctclient's timeout or just as an arbitrary large timeout?

Reply:
This just copies the same initialization fields as the underlying client that the publisher CT client uses.